### PR TITLE
Expose JavaMilli to the whole sbt package

### PR DESCRIPTION
**Pull request by dwijnand**
Allows for reuse, such as in sbt/sbt#4067.

### CODE REVIEW

**Comment by typesafe-tools** [comment]
To validate this pull request against other sbt modules, please visit <A HREF="https://jenkins.scala-sbt.org:8592/job/sbt-validator/parambuild/?sbt=1.1.x&zinc=1.1.x&io=pull/139/head&librarymanagement=1.1.x&util=1.1.x&website=1.1.x&deploy_to_bintray=false&run_sbt_community=false&verbose_dbuild=false&community_verbose=false">this link.</A>

File: io/src/main/scala/sbt/internal/io/Milli.scala
Comment (eed3si9n) [inline io/src/main/scala/sbt/internal/io/Milli.scala:56]: I am of two minds on this one.
I thought we discussed this type of things previously and said we should keep internal code to each modules, and essentially copy-paste so they are less coupled.

Comment (cunei) [reply io/src/main/scala/sbt/internal/io/Milli.scala:56]: I see nothing wrong in exposing internal code to `[sbt]`, it is still confined. Code duplication, however, is almost never a good idea.
This PR has my LGTM

**Review by cunei — DISMISSED** [review]

File: io/src/main/scala/sbt/internal/io/Milli.scala
Comment (dwijnand) [reply io/src/main/scala/sbt/internal/io/Milli.scala:56]: I don't mind making the whole thing public either.

Comment (dwijnand) [reply io/src/main/scala/sbt/internal/io/Milli.scala:56]: by "whole thing" I mean `JavaMilli`, btw.

Comment (eed3si9n) [reply io/src/main/scala/sbt/internal/io/Milli.scala:56]: Making it public is better I think. Let's move it to sbt.io package.

**Comment by dwijnand** [comment]
done. please re-review @cunei and @eed3si9n.

**Review by eed3si9n — DISMISSED** [review]
LGTM pending Travis

**Review by cunei — APPROVED** [review]
Looks ok!

**Pull request merged**
